pandas/core/window.py

```diff
@@ -1044,6 +1058,10 @@ def validate(self):
         elif self.window < 0:
             raise ValueError("window must be non-negative")
 
+        if not self.is_datetimelike and self.closed is not None:
```

Comment:
@jreback here we enforce that `closed` must be `None` for windows that are not datetimelike